docs/geo_upload.md

```diff
@@ -7,7 +7,63 @@ Create metadata submission spreadsheet
 ```
 python geo_metadata.py -m local -d <Dataset_accession>
 ```
-	This will create three files
-	- \<Dataset_accession>\_metadata.csv: This is a csv of all the study, sample, protocol, and fasta file metadata.
-	- \<Dataset_accession>\_md5sum.csv: This is a csv of all the fasta and raw matrix file names and checksums.
-	- \<Dataset_accession>\_s3_uri.csv: This is a list of the s3 URI for all fastq and raw matrix files.
+- This will create three files:
+	- <Dataset_accession>_metadata.csv: This is a csv of all the study, sample, protocol, and fasta file metadata.
+	- <Dataset_accession>_md5sum.csv: This is a csv of all the fasta and raw matrix file names and checksums.
+	- <Dataset_accession>_s3_uri.csv: This is a list of the s3 URI for all fastq and raw matrix files.
+- Upload the metadata and md5sum csv files in a google spreadsheet. Have wrangler add any additional contributor or protocol information.
+
+Launch EC2
+----------------
+- Launch an EC2 instance with enough disk space for 2 upload processes running in parallel.
+- Install c compiler:
+```
+sudo yum install gcc
+```
+- Install ncftp on machine:
+```
+wget ftp://ftp.ncftp.com/ncftp/ncftp-3.2.6-src.tar.gz
+tar -zxvf ncftp-3.2.6-src.tar.gz
+cd ncftp-3.2.6
+./configure
+make
+make install
+make clean
+```
+
+Upload fastq files
+----------------
+- Make sure all raw matrix and fastq files listed have unique files names.
+- Have the wrangler obtain user, password, and directory from GEO. Create subdirectories for raw matrix files and raw sequence files.
+- Create a shell script upload_fastq.sh to download file from s3, upload to GEO, and remove the file locally with following example:
```

Comment:
Looks like there's an additional step of some things to replace in the script below - user, password, directory? Would be handy to list them since it's not super obvious where placeholders are